schematic/visualization/tangled_tree.py

```diff
@@ -243,6 +243,25 @@ def _remove_unwanted_characters_from_conditional_statement(
             cond_req = cond_req_new.replace("If", "").lstrip().rstrip()
         return cond_req
 
+    def _extract_content_within_quotes(self, strings: list[str]) -> list[str]:
+        """
+        Extract content within quotes from a list of strings.
+
+        Args:
+            strings (list): list of strings
+
+        Returns:
+            list: list of strings with content within single quotes
+        """
+        pattern = r"\'(.*?)\'"
```

Comment:
Do we need to account for double quotes?

Reply:
Not from all of the tests that ran, they were all consistent in that the content was all within an escaped single quote, like: `['[\'Diagnosis is "Cancer"\']', '[\'Diagnosis is "Cancer"\']']`


If I go further up in the code where the list is originally created:
```
            conditional_requirements = list(
                attributes_df[
                    (attributes_df["Cond_Req"])
                    & (attributes_df["Component"] == component_name)
                ]["Conditional Requirements"]
            )
```

It seems like this is how pandas is extracting this content out.

Reply:
I swapped to using `ast.literal_eval`: https://docs.python.org/3/library/ast.html#ast.literal_eval

I am using it within the synapse python client for a similar purpose when translating CSV content into python types.